tests/ConsumerTest.php

```diff
@@ -57,7 +57,7 @@ public function testEmitsConsumeEvent()
         $this->dispatcher->expects($this->at(2))->method('dispatch')
             ->with('bernard.acknowledge', new EnvelopeEvent($envelope, $queue));
 
-        $this->assertTrue($this->consumer->tick($queue));
+        self::assertTrue($this->consumer->tick($queue));
```

Comment:
Sebastian Bergmann admits it was a mistake to make `assert*` methods static. Why would you enforce it like this? IMHO makes sense to at at least split it to 2 PRs. `assert*` update is opinionated while the rest is no-brainer.